<ui version="4.0" >
 <class>ossimPlanetQtLayerOperation</class>
 <widget class="QDialog" name="ossimPlanetQtLayerOperation" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Layer Operation</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="theOperationTypeGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Operation Type</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QComboBox" name="theOperationType" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="theParameterGroupBox" >
     <property name="title" >
      <string>Parameter</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QStackedWidget" name="theParameterStack" >
        <property name="currentIndex" >
         <number>1</number>
        </property>
        <widget class="QWidget" name="Opacity" >
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLabel" name="theOpacityReferenceLayerLabel" >
                <property name="text" >
                 <string>Reference Layer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="theOpacitySlider" >
              <property name="maximum" >
               <number>100</number>
              </property>
              <property name="value" >
               <number>50</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLabel" name="theOpacityTopLayerLabel" >
                <property name="text" >
                 <string>Top Layer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Swipe" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QSlider" name="theSwipeSlider" >
            <property name="minimum" >
             <number>-1024</number>
            </property>
            <property name="maximum" >
             <number>1024</number>
            </property>
            <property name="singleStep" >
             <number>1</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ChangeDetectionPage" >
         <layout class="QVBoxLayout" />
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="theCloseButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
